doc/user/layouts/partials/head.html

```diff
@@ -75,45 +74,37 @@
 toCSS | fingerprint }}
 <link rel="stylesheet" href="{{ $style.RelPermalink }}" />
 
-{{/* Kapa.ai */}}
-<script
-  async
-  src="https://widget.kapa.ai/kapa-widget.bundle.js"
-  data-website-id="c9f86179-22af-46ef-8196-289a621ef96c"
-  data-project-name="Materialize"
-  data-project-color="#472E85"
-  data-project-logo="https://avatars.githubusercontent.com/u/47674186?s=200&v=4"
-  data-modal-title="Materialize"
-  data-modal-disclaimer="This AI bot is experimental and might provide inaccurate or incomplete answers. Always consult the official Materialize documentation to verify any answers provided here. __To search the docs instead__, switch to the __Search__ mode (upper-right hand corner).
-
-  Interactions with this AI bot may be monitored or recorded. By submitting your information on this form, you agree to our [website terms](https://materialize.com/site-terms-of-service/) and [privacy policy](https://materialize.com/privacy-policy/)."
-  data-modal-example-questions="Why is my query slow?,How do I check my source's status?"
-  data-search-mode-enabled="true"
-  data-modal-override-open-class-search="docs-search-button"
-  data-user-analytics-fingerprint-enabled="true"
-  data-modal-open-on-command-k="true"
-  data-modal-command-k-search-mode-default="true"
-  data-search-result-link-target="_self"
-  data-search-include-source-names='["Documentation Cloud"]'
-  >
+{{/* Algolia DocSearch */}}
```

Comment:
Could we have 3 separate crawlers? that way the search results don't show answers from all 3 versions of our docs?

- 1 crawler that crawls materialize.com/docs - but excludes self-managed/*
- 1 crawler that crawls materialize.com/docs/self-managed/v25.2
- 1 crawler that crawls  materialize.com/docs/self-managed/v25.1

That way:
- `main` branch use the cloud docs crawler
- `self-managed-docs/v25.2` branch uses the v25.2 crawler
- `self-managed-docs/v25.1` branch uses the v25.1 crawler